schemas/IDLs/org.gel.models.report.avro/4.1.0/ExitQuestionnaire.avdl

```diff
@@ -0,0 +1,297 @@
+@namespace("org.gel.models.report.avro")
+/**
+This protocol defines ExitQuestionnaires
+*/
+protocol ExitQuestionnaires {
+
+    enum CaseSolvedFamily {yes, no, partially, unknown}
+    enum PhenotypesSolved {yes, no, partially, unknown}
+    enum SegregationQuestion {yes, no}
+    enum ReportingQuestion {yes, no, na}
+    enum ConfirmationDecision {yes, no, na}
+    enum ConfirmationOutcome {yes, no, na}
+    enum Actionability {yes, no, not_yet, na}
+    // TODO: refactor with VariantClassification in CommonInterpreted.avdl
+    enum ACMGClassification {pathogenic_variant, likely_pathogenic_variant, variant_of_unknown_clinical_significance, likely_benign_variant,  benign_variant, not_assessed}
+    enum ClinicalUtility {none, change_in_medication, surgical_option, additional_surveillance_for_proband_or_relatives, clinical_trial_eligibility, informs_reproductive_choice, unknown, other}
+
+
+
+
+    record FamilyLevelQuestions{
+        /**
+        Have the results reported here explained the genetic basis of the family’s presenting phenotype(s)?
+        */
+        CaseSolvedFamily caseSolvedFamily;
+        /**
+        Have you done any segregation testing in non-participating family members?
+        */
+        SegregationQuestion segregationQuestion;
+        /**
+        Comments regarding report
+        */
+        string additionalComments;
+    }
+
+    record VariantLevelQuestions{
+        /**
+        Chr: Pos Ref > Alt
+        */
+        string variantDetails;
+        /**
+        Did you carry out technical confirmation of this variant via an alternative test?
+        */
+        ConfirmationDecision confirmationDecision;
+        /**
+        Did the test confirm that the variant is present?
+        */
+        ConfirmationOutcome confirmationOutcome;
+        /**
+        Did you include the variant in your report to the clinician?
+        */
+        ReportingQuestion reportingQuestion;
+        /**
+        What ACMG pathogenicity score (1-5) did you assign to this variant?
+        */
+        ACMGClassification acmgClassification;
+        /**
+        Please provide PMIDs for papers which you have used to inform your assessment for this variant, separated by a ; for multiple papers
+        */
+        string publications;
+
+    }
+
+    record VariantGroupLevelQuestions{
+
+        int variantGroup;
+        array<VariantLevelQuestions> variantLevelQuestions;
+        /**
+        Is evidence for this variant/variant pair sufficient to use it for clinical purposes such as prenatal diagnosis or predictive testing?
+        */
+        Actionability actionability;
+        /**
+        Has the clinical team identified any changes to clinical care which could potentially arise as a result of this variant/variant pair?
+        */
+        array<ClinicalUtility> clinicalUtility;
```

Comment:
Having an array of this enum having values of `none` and `unknown` means we need to check on the application layer that no combinations like `none` and `surgical_option` are allowed. Wouldn't it be easier not to have these "null" values?